components/brave_vpn/brave_vpn_service_desktop.cc

```diff
@@ -429,19 +424,58 @@ void BraveVpnServiceDesktop::LoadCachedRegionData() {
   }
 }
 
+void BraveVpnServiceDesktop::OnCredentialSummary(
+    const std::string& summary_string) {
+  std::string summary_string_trimmed;
+  base::TrimWhitespaceASCII(summary_string, base::TrimPositions::TRIM_ALL,
+                            &summary_string_trimmed);
+  if (summary_string_trimmed.length() == 0) {
+    // no credential found; person needs to login
+    LOG(ERROR) << "No credential found; user needs to login!";
+    SetPurchasedState(PurchasedState::NOT_PURCHASED);
+    return;
+  }
+
+  LOG(ERROR) << "credential_summary returned: `" << summary_string << "`";
```

Comment:
just making sure this won't contain actual credentials

Reply:
This file was removed and logging was updated too (just shows if credential found, not found; doesn't log output)
https://github.com/brave/brave-core/blob/4f0075591f68be23801f21814f8a68abeffb31fb/components/brave_vpn/brave_vpn_service.cc#L1073-L1108

Credential Summary doesn't contain actual credentials; only if there are any. For more info, check out the spec (linked to on security review) and search for `Example responses` where you can check out an example payload 😄